modules/fuse-box-responsive-api/index.js

```diff
@@ -9,10 +9,11 @@
 
     /* @if universal */
     var isBrowser = typeof window !== "undefined";
+    /* @if !npm */
     if (!isBrowser) {
```

Comment:
well here, npm means `no browser` by default, or universal has to be there? Because `npm` should be like extended target `server`, but not `universal`